README.md

```diff
@@ -6,157 +6,90 @@
-
----
+TiKV ('Ti' stands for titanium) is an open-source, distributed, and transactional key-value database with ACID compliance.
 
 With the implementation of the Raft consensus algorithm in Rust and consensus state stored in RocksDB, TiKV guarantees data consistency. [Placement Driver (PD)](https://github.com/pingcap/pd/), which is introduced to implement auto-sharding, enables automatic data migration. The transaction model is similar to Google's Percolator with some performance improvements. TiKV also provides snapshot isolation (SI), snapshot isolation with lock (SQL: `SELECT ... FOR UPDATE`), and externally consistent reads and writes in distributed transactions.
 
 TiKV has the following key features:
 
-- **Geo-Replication**
+- **Geo-Replication**: TiKV uses [Raft](http://raft.github.io/) and the Placement Driver to support Geo-Replication.
+- **Horizontal scalability**: With PD and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale to 100+ TBs of data.
+- **Consistent distributed transactions**: Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
+- **Coprocessor support**: Similar to HBase, TiKV implements a coprocessor framework to support distributed computing.
+- **Cooperates with [TiDB](https://github.com/pingcap/tidb)**: Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transactions, support for RDBMS, and NoSQL design patterns.
 
-    TiKV uses [Raft](http://raft.github.io/) and the Placement Driver to support Geo-Replication.
+![cncf_logo](images/cncf.png)
 
-- **Horizontal scalability**
+TiKV is a graduated project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF). If you are an organization that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how TiKV plays a role, read the CNCF [announcement](https://www.cncf.io/announcements/2020/09/02/cloud-native-computing-foundation-announces-tikv-graduation/).
 
-    With PD and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale to 100+ TBs of data.
+## TiKV Adopters
 
-- **Consistent distributed transactions**
+You can view the list of [TiKV Adopters](https://tikv.org/adopters/).
 
-    Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
+## Quick Start
 
-- **Coprocessor support**
+The most quickest to try out TiKV is using TiUP:
```

Comment:
```suggestion
The quickest way to try out TiKV is using TiUP, an automation program for downloading, updating, and uninstalling TiDB/TIKV components:
```

Reply:
I intentionally removed the description of TiUP since it's already famous enough, how about only leaving a link to TiUP?